portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,27 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const funFacts = [
+    'I perform on an improv comedy team at Purdue.', 
+    'I also have a hydroponic garden.', 
+    'My favorite cereal is Cheerios.', 
+    'When I was in third grade I refused to wear anything other than turtle-neck shirts.',
+  ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // determine the current fact
+  const funFactContainer = document.getElementById('fun-fact-container');
+  const currentFactIndex = funFacts.indexOf(funFactContainer.innerText);
```

Comment:
This is functional but a little inefficient, what if instead of comparing the index down on line 32 you compare the actual text?

Reply:
Updated, thanks!